packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -174,6 +178,33 @@ const ORPHANED_VDI_COLUMNS = [
   },
 ]
 
+const AttachedVdisActions = [
+  {
+    handler: disconnectVbds,
+    icon: 'disconnect',
+    individualHandler: disconnectVbd,
+    individualLabel: _('vbdDisconnect'),
+    label: _('vbdsDisconnect'),
+    level: 'danger',
+  },
+  {
+    handler: deleteVbds,
```

Comment:
This should look like:
`selectedItems => deleteVbds(map(selectedItems, 'vdi'))`

Reply:
why `map(selectedItems, 'vdi')` ?  we want to delete `vbds` not `vdis`

Reply:
Yep. I meant `map(selectedItems, 'vbd')`, my bad. But you also have the `vbd.id` here. Ignore all my comments about VBDs, I saw that you added `vbd.id` afterwards ;)